mantle/kola/harness.go

```diff
@@ -323,10 +327,20 @@ func parseDenyListYaml() error {
 
 	stream := manifest.AddCommitMetadata.FcosStream
 	arch := system.RpmArch()
+	today := time.Now()
 
 	// Accumulate patterns filtering by stream and arch
 	plog.Debug("Processing denial patterns from yaml...")
 	for _, obj := range objs {
+		// Get snooze date
+		snoozeDate, err := time.Parse(snoozeFormat, obj.SnoozeDate)
+		// Ignore snooze date if it was not provided
+		if len(obj.SnoozeDate) > 0 && err != nil {
+			return err
+		} else if len(obj.SnoozeDate) > 0 && today.After(snoozeDate) {
+			continue
+		}
+
```

Comment:
```suggestion
        if obj.SnoozeDate != "" {
    		snoozeDate, err := time.Parse(snoozeFormat, obj.SnoozeDate)
		    if err != nil {
    			return err
	    	} else if today.After(snoozeDate) {
			    continue
		    }
        }

```

Reply:
(OK well the whitespace got corrupted)

Reply:
No worries I get the idea :) This is a lot more readable, thanks for the feedback!